packages/uniforms-bootstrap4/src/LongTextField.tsx

```diff
@@ -16,6 +16,7 @@ function LongText(props: LongTextFieldProps) {
     <textarea
       className={classnames(props.inputClassName, 'form-control', {
         'is-invalid': props.error,
+        'is-valid': !props.error && !!props.value && props.changed,
```

Comment:
Same, the logic for the validity differs in a lot of places:

`!props.error && !!props.value && props.changed`

`!props.error && props.value !== undefined && props.changed`

`!props.error && props.changed`

Is there a reason for this?

Reply:
I tried to not stylize empty fields depending on their value type, but now I realized that `!error && changed` is probably all I wanted to check.